src/main/webapp/pages/samples/sample_files.html

```diff
@@ -6,64 +6,18 @@
 	<title th:text="#{samples.files.title(${sample.label})}">THIS IS SOMETHING WRONG</title>
 	<link rel="stylesheet" th:href="@{/dist/css/sample-files.bundle.css}">
 	<script th:inline="javascript">
-		var PAGE = {
-			sample: {
-				id: /*[[${sample.getId()}]]*/ '1'
-			},
-		  	URLS: {
-			  upload: /*[[@{/samples/{id}/sequenceFiles/upload(id=${sample.getId()})}]]*/ ''
-			}
+		window.PAGE = {
+			id: /*[[${sample.getId()}]]*/ '1',
+			__cm: /*[[${canManageSample}]]*/ false
```

Comment:
Can we have a full variable name here instead of a shorthand?

Reply:
I gave it an shorthand name so that if someone was looking at the code they would not know exactly what it was.  But I can definitely give it a full name.

Reply:
That's a good strategy.  Need to keep keep things confusing.